.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,54 @@
+<!--
+Use Markdown language for styling.
+Github's Markdown Guide https://guides.github.com/features/mastering-markdown/
+Please use the SEARCH function before opening a new issue!
+-->
+
+### --Issue Summary--
+<!-- A brief overview of your issue  -->
+
+### --Video or Image Reference--
+<!-- (Optional) A screenshot, short video, etc. to show your problem -->
+
+### --Steps to reproduce--
+<!-- Describe the steps to reproduce the problem, more steps are better. -->
+
+
+### --System Information--
++ OpenToonz Version:
+<!-- You can find the VERSION in Menu->Help->About OpenToonz -->
+
++ Operating System:
+<!-- (e.g., Windows10, macOS 10.12, Ubuntu 16.04, Debian Stretch) -->
+
++ RAM Size:
+<!-- (4GB, 8GB etc.) -->
```

Comment:
In my experience it was rare that the issue was proved to be due to the memory shortage so far.
How about making this item optional, or combining with the following `Graphics Tablet` item and rename like `Other Device Info` which is for users to write down any optional device information?